lambda/src/main/resources/db/migration/V8__add-matches-table

```diff
@@ -0,0 +1,38 @@
+DROP TABLE "FFIDMetadata";
+CREATE TABLE "FFIDMetadata"
+(
+    "FFIDMetadataId" uuid NOT NULL,
+    "FileId" uuid NOT NULL,
+    "Software" text COLLATE pg_catalog."default" NOT NULL,
+    "SoftwareVersion" text COLLATE pg_catalog."default" NOT NULL,
+    "Datetime" timestamp with time zone not null,
+    "BinarySignatureFileVersion" text COLLATE pg_catalog."default" NOT NULL,
+    "ContainerSignatureFileVersion" text COLLATE pg_catalog."default" NOT NULL,
+    "Method" text COLLATE pg_catalog."default" NOT NULL,
+    CONSTRAINT "FFIDMetadata_pkey" PRIMARY KEY ("FFIDMetadataId"),
+    CONSTRAINT "FFIDMetadata_Consignment_fkey" FOREIGN KEY ("FileId")
+        REFERENCES "File" ("FileId") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+        NOT VALID
+)
+WITH (
+    OIDS = FALSE
+);
+
+CREATE TABLE "FFIDMetadataMatches"
+(
+    "FFIDMetadataId" uuid NOT NULL,
+    "Extension" text COLLATE pg_catalog."default",
+    "IdentificationBasis" text COLLATE pg_catalog."default" NOT NULL,
+    "PUID" text COLLATE pg_catalog."default",
+    CONSTRAINT "FFIDMetadataMatches_Metadata_fkey" FOREIGN KEY ("FFIDMetadataId")
+        REFERENCES "FFIDMetadata" ("FFIDMetadataId") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+        NOT VALID
+)
+WITH (
+    OIDS = FALSE
```

Comment:
Do we need to remember to add this to every table creation? The [Postgres docs](https://www.postgresql.org/docs/8.1/datatype-oid.html) suggest this is the default:

> OIDs are not added to user-created tables, unless `WITH OIDS` is specified when the table is created, or the `default_with_oids` configuration variable is enabled.

Reply:
I guess not based on that. I've never used postgres before here so I've just been copying what Phil wrote. It works fine without it so I've taken it out.

Reply:
Cheers. I was worried it was something we were going to have to keep remembering to do. 👍